browser.py

```diff
@@ -23,6 +23,7 @@
 from gettext import gettext as _
 from base64 import b64decode, b64encode
 
+gi.require_version('Soup', '3.0')
```

Comment:
`import gi` is needed before calling `gi.require_version`

Reply:
Thanks, amended.